content/documentation/pages/5-feature-guides/1-streams/11-monitoring.md

```diff
@@ -121,7 +121,8 @@ This section describes how to set up Prometheus and InfluxDB for Kubernetes.
 
 ### Prometheus
 
-Prometheus is a popular pull-based time series database that pulls metrics from the target applications from a pre-configured endpoint. When running in Kubernetes, Prometheus "scrapes" metrics from target applications that have a specific pod-level annotation. The endpoint to scrape is provided by Spring Boot, under the default path of `/actuator/prometheus`.
+Prometheus is a popular pull-based time series database that pulls metrics from the target applications from a pre-configured endpoint.
+Data Flow leverages the [Prometheus RSocket](https://github.com/micrometer-metrics/prometheus-rsocket-proxy) to establish a persistent, bidirectional `RSocket` connections between all Stream and Task applications and one or more `Prometheus RSocket Proxy` instances. Prometheus is configured to scrape each proxy instance. Proxies in turn use the connection to pull metrics from each application. The scraped metrics are viewable through Grafana dashboards.
```

Comment:
Should the new-lines be the next line like in previous section?

Reply:
Not sure what do you mean?  
Mind that new-line in the MD is not a new  line in the generated doc. An empty new line in the MD is required to produce an actual new line.